uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/directives/leave-request-popup.js

```diff
@@ -36,7 +36,8 @@ define([
           contactId: '<',
           leaveRequest: '<',
           leaveType: '@',
-          userRole: '@'
+          userRole: '@',
+          selectedContactId: '<'
```

Comment:
`contactId` - currently logged in user
`selectedContactId` - currently being browsed contact (in case of admin)
`adminId` - I need to get in CRM.adminId > absence-tab-container.js > absence-tab-container.html > leave-request-popup call > leave-request-popup.js > request-ctrl.js as `contactId` - changed it to contactId.

However, I share your confusion, I would use:
```js
userId // currently logged in user
contactId // currently being browsed contact
```